pkgs/development/ocaml-modules/ocamlformat/ocamlformat.nix

```diff
@@ -2,11 +2,7 @@
 
 with ocaml-ng.ocamlPackages;
```

Comment:
this shouldn't open `ocaml-ng.ocamlPackages`. It's still using the default OCaml version. 

Instead, the library dependencies should be passed as arguments.

Reply:
Good catch thanks!

This bug was here before and made the library incompatible when using a non-default version. Fixed!